### PR TITLE
Fix invalid plane name

### PR DESCRIPTION
**Pull request by EarlMilktea**
Fix invalid plane index `ZX`.

### CODE REVIEW

File: tests/test_gflow.py [hunk 1]
```diff
@@ -156,7 +156,7 @@ def _graph5() -> GraphForTest:
     graph = nx.Graph()
     graph.add_nodes_from(nodes)
     graph.add_edges_from(edges)
-    meas_planes = {0: "XY", 1: "XY", 2: "ZX", 3: "YZ"}
+    meas_planes = {0: "XY", 1: "XY", 2: "XZ", 3: "YZ"}
```
Comment (thierry-martinez) [inline tests/test_gflow.py:159]: This error should have been caught by an AssertionError or another exception.

**Review by thierry-martinez — CHANGES_REQUESTED** [review]
This error should have been caught by an `AssertionError` or another exception.

File: tests/test_gflow.py (hunk 1 above)
Comment (EarlMilktea) [reply tests/test_gflow.py:159]: I totally agree. Let's use pydantic.

Comment (shinich1) [reply tests/test_gflow.py:159]: Note: data validation to `find_gflow` and others being added in #155 by @thierry-martinez , based on new data classes

Comment (shinich1) [reply tests/test_gflow.py:159]: see below by @thierry-martinez 
https://github.com/TeamGraphix/graphix/pull/155/commits/778f709ecf45b8f9bc4f83d3fb0a1b977b0e886b

**Comment by shinich1** [comment]
closing because this was fixed with added plane validation routine in #155.

**Pull request closed without merging**